### PR TITLE
Document rotating AWS IAM Keys

### PR DESCRIPTION
**Pull request by erino**
Document rotating AWS IAM keys

Rendered 👉  https://github.com/alphagov/govuk-developer-docs/blob/key-rotation/source/manual/aws-iam-key-rotation.html.md or https://govuk-developer-docs-f-pr-2090.herokuapp.com/manual/aws-iam-key-rotation.html

### CODE REVIEW

File: source/manual/alerts/rotate-old-aws-iam-key.html.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+owner_slack: "#govuk-2ndline"
+title: Rotate old AWS IAM Key
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-11-04
+review_in: 6 months
+---
+
+GOV.UK uses AWS [IAM](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_users.html)
```
Comment (issyl0) [inline source/manual/alerts/rotate-old-aws-iam-key.html.md]: Do we have to have an alerts page, or can we just link from Icinga to the manual page for how to do it? This feels like unnecessary duplication.

File: source/manual/aws-iam-key-rotation.html.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+in production systems this could prevent a minor outage scaling into an
+incident.
+
+Notify [2ndline](/manual/2nd-line.html) before deleting or making any production
+key inactive.
+
+## Command Line Interface
+
+It's also possible to view and update access keys with the
+[AWS ClI](https://aws.amazon.com/blogs/security/how-to-rotate-access-keys-for-iam-users/).
```
Comment (issyl0) [inline source/manual/aws-iam-key-rotation.html.md]:
```suggestion
[AWS CLI](https://aws.amazon.com/blogs/security/how-to-rotate-access-keys-for-iam-users/).
```

Comment (erino) [reply source/manual/aws-iam-key-rotation.html.md]: Good spot

Comment (erino) [reply source/manual/aws-iam-key-rotation.html.md]: Fixed in f31e5d0

File: source/manual/alerts/rotate-old-aws-iam-key.html.md (hunk 1 above)
Comment (erino) [reply source/manual/alerts/rotate-old-aws-iam-key.html.md]: I think we can link directly to the manual, but as every other alert links to a check I was following the convention...

Comment (erino) [reply source/manual/alerts/rotate-old-aws-iam-key.html.md]: It's possible to directly link to the docs https://github.com/alphagov/govuk-puppet/pull/9820

**Comment by erino** [comment]
@cbaines thanks, there's be plenty of time for testing and improvement as more keys are rotated!

**Pull request merged**